en/docs/develop/identity-governance-rest-api.md

```diff
@@ -3,16 +3,164 @@ template: templates/swagger.html
 ---
 
 # WSO2 Identity Server - governance connectors admin API
+     
+This is the RESTful API for managing identity governance configurations in WSO2 Identity Server. These configurations
+ are grouped in to main 5 categories. Each category will have sub categories which are called connectors, and 
+ properties of connectors as mentioned below. <br>
 
-??? Note "Note: Click to view"
+??? "Password Policies"
+
+    * Password History
+    
+    | Property display name | Property name | Purpose |
+    | ------- | -------|---|
+    | Enable Password History Feature   | passwordHistory.enable   | Enable to disallow previously used passwords |
+    | Password History validation count   | passwordHistory.count   | Restrict reusing last x number of password during password update |
+    
+    * Password Patterns
+    
+    | Property display name | Property name | Purpose |
+    | ------- | -------|---|
+    | Enable Password Policy Feature   | passwordPolicy.enable   | Enable password pattern policy |
+    | Password Policy Min Length   | passwordPolicy.min.length   | Minumum length of the password |
+    | Password Policy Max Length   | passwordPolicy.max.length   | Maximum length of the password |
+    | Password Policy Pattern   | passwordPolicy.pattern   | Allowed password regex pattern |
+    | Password Policy Error Message   | passwordPolicy.errorMsg   | Error message for invalid password patterns |
+
+??? "Login Policies"
```

Comment:
The format for collapsible is-  ??? note "xxx". Please change in all relevant places

Reply:
Without adding the word "note" it is rendered as a collapsible. So not added the word.